portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,17 +33,16 @@
 import com.google.appengine.api.users.UserServiceFactory;
 
 
-/** Servlet responsible for creating new comments. */
+/** Servlet responsible for creating new comments and returning all comments. Max number of comments can be set by user. */
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-  private ArrayList<String> comments = new ArrayList<>(); 
   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private int maxComments = 10;
```

Comment:
This shouldn't be an instance variable, otherwise you'll get different behavior on consecutive requests.

/data --> returns 10 comments
/data?max-comments=5 --> returns 5 comments
/data --> returns 5 comments, even though it should return the default number of comments

Reply:
I've removed it!